azure-pipelines.release.yml

```diff
@@ -56,7 +56,7 @@ resources:
     - repository: templates
       type: github
       name: hmcts/azure-devops-templates
-      ref: refs/heads/master
+      ref: refs/heads/dotnet6-test-branch
```

Comment:
I dont think we should be changing this? Is there something being tested thats not in the master branch? Perhaps we would want this repo to get merged first then?

Reply:
yeah, we shouldn't need to change this file